org.kde.labplot2.json

```diff
@@ -0,0 +1,144 @@
+
+    "modules": [
+        {
+            "name": "gsl",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.gnu.org/gnu/gsl/gsl-2.6.tar.gz",
+                    "sha256": "b782339fc7a38fe17689cb39966c4d821236c28018b6593ddb6fd59ee40786a8"
+                }
+            ]
+        },
+        {
+            "name": "hdf5",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.10/hdf5-1.10.7/src/hdf5-1.10.7.tar.gz",
+                    "sha256": "7a1a0a54371275ce2dfc5cd093775bb025c365846512961e7e5ceaecb437ef15"
+                }
+            ]
+        },
+        {
+            "name": "netcdf",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/Unidata/netcdf-c/archive/v4.7.4.tar.gz",
+                    "sha256": "99930ad7b3c4c1a8e8831fb061cb02b2170fc8e5ccaeda733bd99c3b9d31666b"
+                }
+            ]
+        },
+        {
+            "name": "cfitsio",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-3.49.tar.gz",
+                    "sha256": "5b65a20d5c53494ec8f638267fca4a629836b7ac8dd0ef0266834eab270ed4b3"
+                }
+            ]
+        },
+        {
+            "name": "qtmqtt",
+	    "buildsystem": "qmake",
+	    "builddir": true,
+	    "make-install-args": ["INSTALL_ROOT=/app/qtmqtt"],
+	    "sources": [ { "type": "git", "url": "https://github.com/qt/qtmqtt.git",
+                           "commit": "0da764d8fec459ee28b70f68028ada81ed31abf7", "tag": "v5.15.1" } ]
+	},
+        {
+            "name": "libcerf",
+	    "buildsystem": "cmake-ninja",
+	    "builddir": true,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://jugit.fz-juelich.de/mlz/libcerf/-/archive/v1.14/libcerf-v1.14.tar.gz",
+                    "sha256": "065346b3360943c9961517f8c49ae13fe956835f6fc3b53e9d307e41feec3a34"
+                }
+            ]
+        },
+        {
+            "name": "kuserfeedback",
+	    "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/kuserfeedback/kuserfeedback-1.0.0.tar.xz",
+                    "sha256": "5a2f53ebb4b99a280757ca32bd9b686a7764a726e7e4d8bafee33acbb44b9db7"
+                }
+            ]
+        },
+        {
+            "name": "cantor",
+            "buildsystem": "cmake-ninja",
+            "sources": [ { "type": "git", "url": "https://anongit.kde.org/cantor.git",
+                           "commit": "0c46c2b6b231373f0dce9a46dd6acd90fc01ad33", "tag": "v20.11.80" } ],
```

Comment:
Does it really need the unstable version of cantor?

Also in general i'd prefer if you could use tarballs over git commits/tags